h/static/scripts/annotator/anchoring/test/pdf-test.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var TextQuoteAnchor = require('dom-anchor-text-quote');
+
+var FakePDFViewerApplication = require('./fake-pdf-viewer-application');
+var pdfAnchoring = require('../pdf');
+
+/**
+ * Return a DOM Range which refers to the specified `text` in `container`.
+ *
+ * @param {Element} container
+ * @param {string} text
+ * @return {Range}
+ */
+function findText(container, text) {
+  return new TextQuoteAnchor(container, text).toRange();
+}
+
+var fixtures = {
+  pdfContent: [
```

Comment:
Not preventing the merge, but it's a good idea to add a comment here indicating that each index is a new page